src/runtime/tests/manifest-test.ts

```diff
@@ -1742,7 +1721,7 @@ recipe SomeRecipe
     const manifestSource = `
         schema Thing
           someProp: Text
-        store Store0 of [Thing] in '${Flags.useNewStorageStack ? 'new-entities.json' : 'entities.json'}'`;
+        store Store0 of [Thing] in 'new-entities.json'`;
```

Comment:
can entities.json be deleted? (or maybe rename new-entities.json to entities.json)